rrule/rrule.d.ts

```diff
@@ -0,0 +1,249 @@
+// Type definitions for rrule 2.1.0
+// Project: https://github.com/jkbrzt/rrule
+// Definitions by: James Bracy <https://github.com/waratuman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'rrule' {
+    export = RRule;
+}
+
+declare namespace RRule {
+
+    /**
+    * see <http://labix.org/python-dateutil/#head-cf004ee9a75592797e076752b2a889c10f445418>
+    * The only required option is `freq`, one of RRule.YEARLY, RRule.MONTHLY, ...
+    */
+    interface Options {
+        freq: any; // Should be enum
+        dtstart?: Date;
+        interval?: number;
+        wkst?: number // | day enum
+        count?: number;
+        until?: Date;
+        bysetpos?: number;
+        bymonth?: number;
+        bymonthday?: number;
+        byyearday?: number;
+        byweekno?: number;
+        byweekday?: Weekday[];
+        byhour?: number;
+        byminute?: number;
+        bysecond?: number;
+    }
+
+    interface Weekday {
+        new (n: number): Weekday;
```

Comment:
As written this means that individual `Weekday` instances are new-able. If `Weekday` itself is new-able, use a class with `constructor(n: number);`. Also, looks like `nth` may be `static`.